mlx/ops.cpp

```diff
@@ -133,6 +154,46 @@ std::pair<int, int> extract_quantized_matmul_dims(
   return {w_inner_dims, w_outer_dims};
 }
 
+std::pair<int, int> extract_qqmm_dims(
+    std::string_view tag,
+    const array& x,
+    const array& w_q,
+    const array& scales_w,
+    const std::optional<array>& w,
+    int group_size,
+    int bits) {
+  // Validate w_q and scales_w
+  validate_quantized_input(
+      tag, w_q, scales_w, "weight matrix", "scales_w", group_size, bits);
```

Comment:
I think you can remove the strings here since `x` is not quantized. The original error message prior to the diff here makes sense.